pkg/service/api/v1/deploy/checking.go

```diff
@@ -270,8 +273,98 @@ func checkWrongClusterConfiguration() (errs []*api.CheckingItem) {
 	return errs
 }
 
+func checkClusterHANodeCount() (warnings []*api.CheckingItem) {
+
+	warnings = make([]*api.CheckingItem, 0)
+	wizardData := wizard.GetCurrentWizard()
+	counters := map[constant.MachineRole]uint{
+		constant.MachineRoleEtcd:    0,
+		constant.MachineRoleMaster:  0,
+		constant.MachineRoleWorker:  0,
+		constant.MachineRoleIngress: 0,
+	}
+	haCount := map[constant.MachineRole]uint{
+		constant.MachineRoleEtcd:    3,
+		constant.MachineRoleMaster:  2,
+		constant.MachineRoleWorker:  2,
+		constant.MachineRoleIngress: 2,
+	}
+	for _, node := range wizardData.Nodes {
+
+		for _, role := range node.MachineRoles {
+			counters[role]++
+		}
+	}
+	for role, counter := range counters {
+
+		if counter >= haCount[role] {
+			continue
+		}
+
+		warnings = append(warnings, &api.CheckingItem{
+			CheckingPoint: fmt.Sprintf("Nodes for %s high available", role),
+			Result:        constant.CheckResultWarning,
+			Error: &api.Error{
+				Reason:     fmt.Sprintf("nodes for %s are not enough for high available", role),                                     // %s 角色的节点数不满足高可用数量
+				Detail:     fmt.Sprintf("We suggest nodes for %s count should be equal or more than %d nodes", role, haCount[role]), // 我们建议 %s 角色节点数大于等于%d 个
+				FixMethods: fmt.Sprintf("Add new node for role: %s, or edit existing node to include this role", role),              // 添加一个新节点包含 %s 角色，或者编辑已有节点使他包含这个角色
+			},
+		})
+	}
+
+	return
+}
+
+func checkClusterNodePortMinimum() (warnings []*api.CheckingItem) {
+
+	warnings = make([]*api.CheckingItem, 0)
+	wizardData := wizard.GetCurrentWizard()
+	if wizardData.Info.NodePortMinimum <= suggestNodePortMinimum {
+
+		warnings = append(warnings, &api.CheckingItem{
+			CheckingPoint: fmt.Sprintf("Node port minimum setting too small warning"), // NodePort 最小端口号设置过小警告
+			Result:        constant.CheckResultWarning,
+			Error: &api.Error{
+				Reason:     "Node port minimum setting too small",                                                             // NodePort 最小端口号太小
+				Detail:     fmt.Sprintf("We suggest node port minimum port should be higher than %d", suggestNodePortMinimum), // 我们建议 NodePort 最小端口号大于 %d
+				FixMethods: fmt.Sprintf("Modify the node port minimum higher than %d", suggestNodePortMinimum),                // 修改最小端口号为大于 %d
+			},
+		})
+	}
+
+	return
+}
+
+func checkClusterNodePortInterval() (warnings []*api.CheckingItem) {
+
+	warnings = make([]*api.CheckingItem, 0)
+	wizardData := wizard.GetCurrentWizard()
+	if wizardData.Info.NodePortMaximum-wizardData.Info.NodePortMinimum > suggestNodePortMaxInterval {
+
+		warnings = append(warnings, &api.CheckingItem{
+			CheckingPoint: "Node port interval setting too large warning", // NodePort 监控区间设置过大警告
```

Comment:
"Node port interval setting too large warning" -> "Warning of too large value on node port interval setting"

Reply:
Fixed